app/src/main/java/com/lagradost/cloudstream3/utils/BackupUtils.kt

```diff
@@ -69,7 +77,12 @@ object BackupUtils {
 
         "biometric_key", // can lock down users if backup is shared on a incompatible device
         "nginx_user", // Nginx user key
-        "download_path_key" // No access rights after restore data from backup
+
+        // No access rights after restore data from backup
+        "download_path_key",
+        "download_path_pref",
```

Comment:
added `download_path_pref` in ignoreForBackup list, check if that is something we left deliberately.

Reply:
also check this value in `strings.xml`, why are we not keeping it `translatable=false`?

Reply:
Looks like a __big__ mistake, download_path_pref is used both as a display string and as a key, it should never be used as both.

Reply:
correct, I would fix it but would let one of the maintainers comment on this.
\